cn.navclub.vdbfx.json

```
@@ -0,0 +1,77 @@
+{
+    "app-id" : "cn.navclub.dbfx",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "v-dbfx",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "libgee",
+            "buildsystem" : "autotools",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libgee.git"
+                }
+            ]
+        },
+        {
+            "name" : "flatpak_mysql",
+            "no-autogen" : true,
+            "buildsystem" : "cmake",
+            "cleanup" : [
+                "lib",
+                "include"
```

```suggestion
                "/lib",
                "/include"
```

But then I'm not sure if removing the libraries will work unless you only produce static library here.